.github/workflows/ci.yaml

```diff
@@ -30,9 +30,9 @@ jobs:
           npm run lint
           npm test
           npm run synth
-      - name: clean, compile, test and upload
+      - name: clean, compile, test and upload facia-purger
```

Comment:
The GitHub Action is only building `facia-purger`. The build of `backup-parameter-store` is still happening in [TeamCity](https://teamcity.gutools.co.uk/viewType.html?buildTypeId=dotcom_BackupParameterStore&branch_dotcom=upgrade-to-scala-2.13&tab=buildTypeStatusDiv)